### PR TITLE
Continuous integration

### PR DESCRIPTION
**Pull request by fujiisoup**
By this PR, fac will support a continuous integration system.
A cloud server (travis: https://travis-ci.org/) will compile and run the following tests automatically and report back the result by the indicator (see the green check mark next to the commit id).

Every push to this repo (including pull requests) will trigger the compilation and testing.

I only add one test for c and a few tests for python (test_li_excitation.py, test_se_ai.py, test_spec.py).
Ideally, these test should check the validness of the result, but currently I just check whether lthe scripts are working without error.

In the long term, it would be nice to enrich more test cases so that we do not accidentally break the code.

### CODE REVIEW

**Comment by fnevgeny** [comment]
In cfac, there is a "make check" rule.

**Comment by fujiisoup** [comment]
Nice. 
Are you checking the compilation status or did you prepared some unit tests?
It would be nice if fac can import this machinery from cfac.

**Comment by fnevgeny** [comment]
It runs test calculations from the "demo" dir and compares the results with confirmed ones.

**Comment by fujiisoup** [comment]
It's nice.
OK. we should have a sImilar machinery.
Checking demo result sounds nice as this would be robust.

File: demo/check.py [hunk 1]
```diff
@@ -0,0 +1,598 @@
...
+    _check_header(actual_header, expected_header)
+    _check_block(actual_blocks, expected_blocks, 
+                 atols={'Delta E': 1.0e0, 'bethe': 1.0e0, 'born': 1.0, 
+                        'collision strength': 0.001, 'crosssection': 0.01,
+                        'TEGRID': 1.0, 'EGRID': 1.0, 'USR': 1.0, 
+                        'TE0': 1.0},
+                 rtols={'Delta E': 0.01, 'bethe': 0.01, 'born': 0.01, 
+                        'collision strength': 0.02, 'crosssection': 0.1,
+                        'TEGRID': 0.01, 'EGRID': 0.01, 'USR': 0.01, 
+                        'TE0': 0.01})
```
Comment (fujiisoup) [inline demo/check.py]: This python script checks the demo calculation with the pre-calculated results in each ref directory.
`atols` and `rtols` are maximum error amplitude that we accept. 
I chose these values intuitively, but if anyone has comments about this, I will improve them.

File: demo/polariz/Makefile [hunk 2]
```diff
@@ -1,4 +1,5 @@
-RESULTS = ta.ce ta.en ta.tr pol.txt pop.txt
```
Comment (fujiisoup) [inline demo/polariz/Makefile]: I notice that `EGRID` in `ta.ce` file calculated by fac=1.1.4 has a large difference from that by fac=1.1.1.
Were any changes made between these two releases?

**Comment by fujiisoup** [comment]
I think this PR is ready.
I'm going to merge this in a few days, but any comments / suggestions would be appreciated.

**Comment by fujiisoup** [comment]
Now tests are automatically running :)

**Pull request merged**